template/template_test.go

```diff
@@ -350,19 +350,23 @@ func checkCVEBody(t *testing.T, body string, custom map[string]interface{}) {
 	t.Helper()
 	assert.True(t, strings.Contains(body, toString(custom["repo_url"])))
 	assert.True(t, strings.Contains(body, toString(custom["scanned_at"])))
+	assert.True(t, strings.Contains(body, toString(custom["total_dependencies"])))
+	dirDepArr := toArrMap(custom["direct_updates"])
+	assert.NotNil(t, dirDepArr)
+	transDepArr := toArrMap(custom["transitive_updates"])
+	assert.NotNil(t, transDepArr)
+
+	checkDepDataCVE(t, body, custom["direct_updates"])
+	checkDepDataCVE(t, body, custom["transitive_updates"])
```

Comment:
you already converted `direct_updates` and `transitive_updates`  using `toArrMap()` in previous line.  You can pass them to `checkDepDataCVE()` function.